### PR TITLE
fix: voting power for offchain pending proposals should use the proposal snapshot

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Closes: #940

This PR fix the voting power for offchain pending proposal, which were based on `latest` block, instead of the proposal own snapshot

### How to test

1. Open the network in the dev console
2. Go to an offchain pending proposal
3. Inspect the payload for `https://score.snapshot.org/`
4. It should use a number as `snapshot`, instead of `latest`

### CODE REVIEW

File: apps/ui/src/composables/useVotingPower.ts [hunk 1]
```diff
@@ -13,8 +13,10 @@ export function useVotingPower() {
 
   function getProposalSnapshot(proposal: Proposal): number | null {
     return (
-      (proposal.state === 'pending' ? null : proposal.snapshot) ||
-      getLatestBlock(proposal.network)
+      (proposal.state === 'pending' &&
+      !offchainNetworks.includes(proposal.network)
+        ? null
+        : proposal.snapshot) || getLatestBlock(proposal.network)
```
Comment (ChaituVR) [inline apps/ui/src/composables/useVotingPower.ts:19]: Too complex to understand.

**Review by ChaituVR — APPROVED** [review]
tAck

File: apps/ui/src/composables/useVotingPower.ts (hunk 1 above)
Comment (wa0x6e) [reply apps/ui/src/composables/useVotingPower.ts]: Is it better ?

Comment (ChaituVR) [reply apps/ui/src/composables/useVotingPower.ts]: Yep :D

**Pull request merged**